dicodile/data/gait.py

```diff
@@ -40,13 +50,96 @@ def get_gait_data(subject=1, trial=1):
     gait_dir.mkdir(parents=True, exist_ok=True)
     gait_zip = download(
         "http://dev.ipol.im/~truong/GaitData.zip",
-        gait_dir / "GaitData.zip"
+        gait_dir / "GaitData.zip",
+        verbose=verbose
     )
 
     with ZipFile(gait_zip) as zf:
         with zf.open(f"GaitData/{subject}-{trial}.json") as meta_file, \
-             zf.open(f"GaitData/{subject}-{trial}.csv") as data_file:
+                zf.open(f"GaitData/{subject}-{trial}.csv") as data_file:
             meta = json.load(meta_file)
-            data = pd.read_csv(data_file, sep=',', header=0)
-            meta['data'] = data
+            if not only_meta:
+                data = pd.read_csv(data_file, sep=',', header=0)
+                meta['data'] = data
             return meta
+
+
+def get_gait_code_list():
+    """Returns the list of all available codes.
+
+    Returns
+    -------
+    list
+        List of codes.
+    """
+    try:
+        with open(GAIT_CODE_LIST_FNAME, "r") as f:
+            code_list = json.load(f)
+
+    except FileNotFoundError:
+        gait_dir = DATA_HOME / "gait"
+        gait_dir.mkdir(parents=True, exist_ok=True)
+        gait_zip = download(
+            "http://dev.ipol.im/~truong/GaitData.zip",
+            gait_dir / "GaitData.zip",
+        )
```

Comment:
You should only have one function that downloads the data. Don't duplicate code needlessly a this makes it harder to maintain it.